packages/smart-file-system/src/SmartFileInfo.php

```diff
@@ -123,8 +123,8 @@ public function getRealPath(): string
         return parent::getRealPath() ?: $this->getPathname();
     }
 
-    private function getNormalizedRealPath(): string
+    private function getNormalizedPath(?string $string = null): string
```

Comment:
Getter cannot accept any argument.

These should be 2 different methods:

- `getNormalizedPath()`
- `normalitePath($path)`

Reply:
Ok, fixed!